### DISCUSSION TITLE
进群发典！

### DISCUSSION THREAD

**Discussion by blqdswkl** [share121/inter-knot#2139] [其他]
![qrcode_1723647360023](https://github.com/user-attachments/assets/52671d51-d83c-451a-9986-a655456df387)
干嘛都行，发图也行，群名群头像随便改，但是不要刷屏骂人聊政治.....其它敬请发挥(

**Comment by MingSun-y** [comment]
打破零回

**Comment by theincrediblewwz** [comment]
兄啊，群问题答案是啥

**Reply by aomengxin** [reply]
我觉得随便填一个就行👀

**Comment by manyshoes** [comment]
这都是什么群啊，你们这个群害人不浅啊<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by CRP123456** [comment]
问题的答案是什么
